composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": "^5.3.9|^7.0",
-        "imagine/Imagine": "^0.6.3,<0.7",
+        "imagine/Imagine": "^0.6.3|^0.7.0,<0.8",
```

Comment:
`^0.7` and `^0.7.0` are actually equivalent. But I understand your point. It is more clear for those who don't know how the the caret operator works.